src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabDetails/components/VirtualMachineStatusWithPopover/VirtualMachineProvisioningStatus.tsx

```diff
@@ -47,7 +44,11 @@ const VirtualMachineProvisioningStatus: FC<VirtualMachineProvisioningStatusProps
           ))}
           <br />
           <Content component="p">
-            <Link to={createURL('diagnostics', location?.pathname)}>{t('View diagnostic')}</Link>
```

Comment:
do we still use createUrl or we can delete it?

Reply:
it's in use:  https://github.com/search?q=repo%3Akubevirt-ui%2Fkubevirt-plugin%20createURL(&type=code